SharpBucketTests/V2/EndPoints/IssueResourceTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework;
+using SharpBucket.V2.EndPoints;
+using SharpBucket.V2.Pocos;
+using Shouldly;
+
+namespace SharpBucketTests.V2.EndPoints
+{
+    [TestFixture]
+    internal class IssueResourceTest
+    {
+        private IssueResource ExistingIssue { get; set; }
+
+        private IssueResource NotExistingIssue { get; set; }
+
+        [OneTimeSetUp]
+        protected void Init()
+        {
+            ExistingIssue = SampleRepositories.BotTestRepository.IssuesResource().IssueResource(2);
+
+            NotExistingIssue = SampleRepositories.BotTestRepository.IssuesResource().IssueResource(int.MaxValue);
```

Comment:
Maybe one tests using this is missing ?

Reply:
What do you mean? The last test in the file, called `ListIssues_NotExistingRepository_ThrowException` uses it.

Reply:
Ok i've miss it